cv32/sim/uvmt_cv32/riviera.mk

```diff
@@ -0,0 +1,297 @@
+###############################################################################
+#
+# Copyright 2020 OpenHW Group
+# 
+# Licensed under the Solderpad Hardware Licence, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     https://solderpad.org/licenses/
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# 
```

Comment:
Please add the following SPDX header at this location:
```
# SPDX-License-Identifier: Apache-2.0 WITH SHL-2.0
```
This is a new requirement we have been asked to add to all headers to enable script-based audits of our code base.